### PR TITLE
Reflections...

### PR DESCRIPTION
**Pull request by zcullisonshimada**
I found the assignment useful in terms of figuring out how to use Markdown and the different operations available to users. I did not, however, find the assignment easy. The most challenging part was navigating the technology. As someone unfamiliar with R and Github, it was difficult figuring out the software and programming, especially when attaching links and images to R Markdown and figuring out how to do what on Github. But now that I've used the material I feel like I know R Markdown and Github better. I feel more comfortable with these platforms. 

I found extremely helpful the Cheat Sheets provided by R Studio. When I still had difficulty figuring out how to generate the codes, I looked up how to do the programming operation on google. When I still had a trouble, I asked a friend to help me out :) In all, I had a good experience.

---
name: PR template for CFSS
about: Used for students submitting a pull request for a homework assignment
title: Description of my submission
labels: ''
assignees: ''

---

Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# hw01
\ No newline at end of file
+# hw01
+title: "Hw 1"
```
Comment (bensoltoff) [inline README.md:2]: If you want these on separate lines, you need a line break between them

```
title: "Hw 1"

author: Zoe Cullison-Shimada  
```

title: "Hw 1"

author: Zoe Cullison-Shimada

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+I'd like to take a philsophy class because I've never taken one before. 
+I'd like to investigate my thoughts and carry out a creative inquiry about the world. Thus, a project of mine  
+  
+*The cultural and philsophical undertones of introductions*  
+
+* forms of introduction 
+  + verbal greetings
+  + visual greetings or eye-contact
+  + hug or handshake  
+A footnote [^1]
```
Comment (bensoltoff) [inline README.md:20]: Footnotes are hard to implement directly in Markdown. You can use them in R Markdown documents and R will basically rewrite the code to make the footnotes actually appear at the bottom of the document. Nice attempt though!

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+A footnote [^3]  
+[^3]: Citations from Cicero  
+
+***  
+# how about an artifact that you're interested in?
+[link](www.cooking.nytimes.com)  
+
+I love _cooking_ and aspire to become a talented chef. _Cooking_ requires creativity, diligence, care, hospitality, and skill...  
+
+![yum](https://dadaeats.com/raw-vegan-gluten-free-blueberry-vanilla-cheesecake/)
```
Comment (bensoltoff) [inline README.md:43]: Was this supposed to be a link or a picture?

File: README.md [hunk 4]
```diff
@@ -1 +1,46 @@
...
+
+* philsophical undertones
+  + hospes or hospitality
+  + kindness
+  + friendship  
+A footnote [^3]  
+[^3]: Citations from Cicero  
+
+***  
+# how about an artifact that you're interested in?
```
Comment (bensoltoff) [inline README.md:38]: It's best practice to leave a blank line between a header and the next content. This will prevent potential rendering issues

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |    x          |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |           |    x          |            |

* Be sure to commit early and often throughout your work. This will be especially important once you start writing code and trying to track changes/revisions

File: README.md (hunk 3 above)
Comment (zcullisonshimada) [reply README.md:43]: A picture, but I was having difficulty.

**Comment by jameskoh98** [comment]
`Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**               |           |              x|            |
| **Coding strategy**         |           x|              |            |
| **Presentation**               |           |              x|            |
| **Achievement, creativity** |         x|              |            |
| **Ease of access**             |           x|              |            |

Remarks:

* Detailed, informative
* As Ben pointed out, some issues with formatting footnotes and the image and link don't work for me but overall a good job

File: README.md [hunk 5]
```diff
@@ -1 +1,46 @@
...
+[link](www.cooking.nytimes.com)
```
Comment (bensoltoff) [inline README.md:39]:
```
[link](www.cooking.nytimes.com)
```

should be

```
[link](https://cooking.nytimes.com)
```

[link](https://cooking.nytimes.com)

You just did not provide a full URL, so GitHub rendered it incorrectly.

File: README.md (hunk 3 above)
Comment (bensoltoff) [reply README.md:43]: Then you need to make sure the URL you use inside of `()` is a link to a picture/image file, not a full web page. For instance, if you want the first image from that page, the syntax is

```
![Yum](https://i1.wp.com/dadaeats.com/wp-content/uploads/2018/10/cheesecake.jpg?w=2000&ssl=1)
```

which appears as

Yum

**Review by jameszimmerman — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |        x      |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:
+ The links and images did not work for me, but otherwise well-done.
+ I particularly liked how you broke your project into a detailed list, using Markdown in order to clarify intent.

**Pull request closed without merging**